tests/hipscat/catalog/association_catalog/test_association_catalog_info.py

```diff
@@ -62,7 +62,11 @@ def test_wrong_type(association_catalog_info_data, catalog_info_data):
     with pytest.raises(TypeError, match="unexpected"):
         AssociationCatalogInfo(**catalog_info_data)
 
-    with pytest.raises(ValueError, match="type association"):
-        init_data = association_catalog_info_data.copy()
-        init_data["catalog_type"] = CatalogType.OBJECT
-        AssociationCatalogInfo(**init_data)
+    with pytest.raises(ValueError, match=f"{CatalogType.ASSOCIATION}"):
+        try:
```

Comment:
Is this try/catch for debugging? Can it be removed?

Reply:
Dang. I thought I removed that. I'll fix it.